test/integration_test/specs/cassandra-clone/appclone.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: stork.libopenstorage.org/v1alpha1
+kind: ApplicationClone
+metadata:
+  name: clonecassandra
```

Comment:
nit: clone-cassandra
You may change if you wish.

Reply:
Updated